tests/security/static/scripts/init.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+REPO_ROOT=$(git rev-parse --show-toplevel)
+BUILD_PATH="$REPO_ROOT/build"
+STATIC_PATH="$REPO_ROOT/tests/security/static"
+CODEQL_CLI_VERSION=$(<"$STATIC_PATH/scripts/codeql.config")
+CODEQL_CLI_PLATFORM_PATH="$BUILD_PATH/tools/codeql-cli/linux"
+CODEQL_CLI_PATH="$CODEQL_CLI_PLATFORM_PATH/$CODEQL_CLI_VERSION"
+CODEQL_CLI_URL=https://github.com/github/codeql-cli-binaries/releases/download/$CODEQL_CLI_VERSION/codeql-linux64.zip
+CODEQL_LIBRARY_PATH="$BUILD_PATH/codeql"
+
+if [ ! -d "$CODEQL_LIBRARY_PATH" ]; then
+    git clone https://github.com/github/codeql.git "$CODEQL_LIBRARY_PATH"
+    cd "$CODEQL_LIBRARY_PATH"
+    # Reset to rc/1.25
+    git reset 589a097197075384a46340a3b3202a3e8ecb3272
```

Comment:
Why does this specifically reset the master instead of simply checking out the appropriate rc/1.25 branch or tag?